src/infra/raft/v2/RaftService.cpp

```diff
@@ -86,9 +87,17 @@ void RaftServer::serverLoopMain() {
 
 //////////////////////////// RaftClient ////////////////////////////
 
-RaftClient::RaftClient(const std::shared_ptr<grpc::Channel> &channel,
-                       uint64_t peerId, EventQueue *aeRvQueue)
-    : mStub(Raft::NewStub(channel)), mPeerId(peerId), mAeRvQueue(aeRvQueue) {
+RaftClient::RaftClient(const std::string &peerAddress,
+                       std::optional<TlsConf> tlsConfOpt,
+                       std::shared_ptr<DNSResolver> dnsResolver,
+                       uint64_t peerId,
+                       EventQueue *aeRvQueue) :
+    mPeerAddress(peerAddress),
+    mDNSResolver(dnsResolver),
+    mTLSConfOpt(tlsConfOpt),
+    mPeerId(peerId),
+    mAeRvQueue(aeRvQueue) {
+  refressChannel();
```

Comment:
maybe we needn't to refresh all clients, only refreshing the connection error nodes is enough.

Reply:
already did in refresh logic